docs/accessibility/patterns-and-contrast.md

```diff
@@ -4,8 +4,33 @@ Patterns and contrast
 The default palette of Highcharts is designed with accessibility in mind, so that any two neighbor colors are tested for different types of color blindness. In addition to that, there are a few ways to increase contrast, both for the visually impaired or for grayscale prints, but also for the charts to be more readable in general.
 
```

Comment:
Suggested additional paragraph here:

Keep in mind that you should not rely on color alone to convey information in the chart if possible. Consider using data labels for points to show the values with numbers, and make sure readers of the chart can understand the data without relying on color.